starfish/experiment/experiment.py

```diff
@@ -18,22 +27,22 @@ class FieldOfView:
     This encapsulates a field of view.  It contains the primary image and auxiliary images that are
     associated with the field of view.
 
-    Auxiliary images can be accessed using a key, i.e., FOV[aux_image_type].
+    All images can be accessed using a key, i.e., FOV[aux_image_type].  The primary image is
+    accessed using the key :py:attr:`starfish.experiment.experiment.FieldOFView.PRIMARY_IMAGE`.
 
     Properties
     -------
-    name                   The name of the FOV.  In an experiment with only a single FOV, this may
-                           be None.
-    primary_image          The primary image for this field of view.
-    auxiliary_image_types  A set of all the auxiliary image types.
+    name         The name of the FOV.
+    image_types  A set of all the image types.
     """
+
+    PRIMARY_IMAGE = 'primary'
```

Comment:
make plurual? PRIMARY_IMAGE -> PRIMARY_IMAGES. in practice, when i call experiment.fov()[PRIMARY_IMAGE] i actually get back a tensor of many images